tools/github-issues/Azure.Sdk.Tools.GitHubIssues/Azure.Sdk.Tools.GitHubIssues.csproj

```diff
@@ -6,16 +6,17 @@
     <NoWarn>$(NoWarn);8002</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Data.AppConfiguration" Version="1.0.1" />
-    <PackageReference Include="Azure.Identity" Version="1.2.0" />
+    <PackageReference Include="Azure.Data.AppConfiguration" Version="1.0.2" />
+    <PackageReference Include="Azure.Identity" Version="1.2.3" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.1.0" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.5.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.6.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.Azure" Version="1.0.0" />
     <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.9" />
     <PackageReference Include="Octokit" Version="0.48.0" />
     <PackageReference Include="Polly" Version="7.2.1" />
-    <PackageReference Include="Sendgrid" Version="9.20.0" />
+    <PackageReference Include="RateLimiter" Version="2.1.0" />
```

Comment:
I couldn't find anything interesting about this package beyond the download on nuget. How did you find it and what exactly is it doing?

Reply:
I basically went shopping for a component that I could easily do a n requests y seconds limit. Which is exactly what this package has in its TimeLimiter type. Its probably a bit over engineered for what we need in that it can have multiple interwoven limits interacting (so for example if you had an API that was subject to two limits ... Azure DevOps and GitHub for example) you can combine those two rate limits together.

For this scenario though I'm just using its basic usage, which is the same thing as I'm planning to do for Check Enforcer.